apps/demo/src/app/counter-rx-mutation/counter-rx-mutation.ts

```diff
@@ -16,6 +16,9 @@ export type CounterResponse = {
   json: { counter: number };
 };
 
+// TODO - rename this file to just be `mutations-functions-standalone` + class/selector etc??
```

Comment:
My pinion + feedback from a test user: I think renaming this and doing a couple other tweaks to the two application examples would be good.